src/store/global/global.thunks.js

```diff
@@ -83,7 +83,15 @@ function fetchFiatExchangeRates (symbols) {
   return (dispatch) => {
     dispatch(globalActions.loadFiatExchangeRates())
 
-    return fiatExchangeRatesApi.getFiatExchangeRates(symbols)
+    return Promise.resolve({
```

Comment:
Wanna add a `TODO` here, so we don't forget to fix it?

Reply:
I'll create an issue